proto/beacon/rpc/v1/services.proto

```diff
@@ -6,36 +6,6 @@ import "google/protobuf/empty.proto";
 import "proto/eth/v1alpha1/beacon_block.proto";
 import "proto/eth/v1alpha1/attestation.proto";
 import "google/api/annotations.proto";
-import "protoc-gen-swagger/options/annotations.proto";
-
```

Comment:
we will not be using swagger in the future ?

Reply:
We don't need to serve swagger locally. It's broken at the moment and no one using it so we can defer to a later date.